prosody/Dockerfile

```diff
@@ -47,13 +47,18 @@ RUN \
     && apt-cleanup \
     && rm -rf /tmp/pkg /var/cache/apt
 
-RUN patch -d /usr/lib/prosody/modules/muc -p0 < /prosody-plugins/muc_owner_allow_kick.patch
 
 COPY rootfs/ /
 
 COPY --from=builder /usr/local/lib/lua /usr/local/lib/lua
 COPY --from=builder /usr/local/share/lua /usr/local/share/lua
 
+ADD https://raw.githubusercontent.com/netaskd/mod_turncredentials/master/mod_turncredentials.lua /prosody-plugins/mod_turncredentials.lua
+
+RUN \
```

Comment:
What relation these lines have with coturn?

Reply:
https://prosody.im/doc/coturn

Reply:
That is the turncredentials. That part is OK.

I'm talking about the
> RUN \
> sed -i s/hook/hook_global/g /prosody-plugins/mod_auth_token.lua \
>   && patch -d /usr/lib/prosody/modules/muc -p0 < /prosody-plugins/muc_owner_allow_kick.patch

That muc_owner_allow_kick.patch is related to turn?

Reply:
No.
It comes from a old commit, refer to 367621f and probably earlier.